Can't precompile HDF5 on OSX

Precompiling `HDF5` works fine on Linux, and I can precompile other packages like `Images` and `Nettle` just fine under OSX, but not `HDF5`.

Using
```
 - BinDeps                       0.3.10
 - Homebrew                      0.1.14             
 - HDF5                          0.4.13
```
I get
```
precompile.jl
error during bootstrap: LoadError(at "sysimg.jl" line 286: LoadError(at "/Users/rene/local/julia/base/userimg.jl" line 1: LoadError(at "/Users/rene/.julia/v0.3/HDF5/src/HDF5.jl" line 1: LoadError(at "/Users/rene/.julia/v0.3/HDF5/src/plain.jl" line 15: LoadError(at "/Users/rene/.julia/v0.3/Homebrew/src/Homebrew.jl" line 353: UndefVarError(var=:STDERR))))))
make[1]: *** [/Users/rene/local/julia/usr/lib/julia/sys0.o] Error 1
make: *** [release] Error 2
```
Versioninfo:
```
Julia Version 0.3.7
Commit cb9bcae* (2015-03-23 21:36 UTC)
Platform Info:
  System: Darwin (x86_64-apple-darwin13.4.0)
  CPU: Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
```
I'd be grateful for suggestions on how to further debug this. Thanks!
cc @staticfloat

--- a/src/plain.jl
+++ b/src/plain.jl
@@ -12,8 +12,6 @@ import Base: close, convert, done, dump, eltype, endof, flush, getindex,
              isempty, isvalid, length, names, ndims, next, parent, read,
              setindex!, show, size, sizeof, start, write
 
-@osx_only import Homebrew # Add Homebrew/lib to the DL_LOAD_PATH
-
 include("datafile.jl")
 
 ## C types

